.github/workflows/deploy.yml

```diff
@@ -46,6 +50,13 @@ jobs:
       url: ${{ steps.deploy-to-webapp.outputs.webapp-url }}
 
     steps:
+      - name: Az CLI login
+        uses: azure/login@v1
+        with:
+          client-id: d1121761-5bcf-486b-aff0-e6451b4f4e05
```

Comment:
Out of interest, where did you register the new app for the client id, etc? Was it through our Primer sub or through the main AD? I'll need to move another repo over to OIDC authentication this week, so would be helpful to know if there are any corresponding PRs I could reference.